src/database/models.py

```diff
@@ -76,44 +79,72 @@ def __str__(self):
 
 
 class UserReview(models.Model):
-    review_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    organization_id = models.ForeignKey(
-        Organization, on_delete=models.CASCADE, to_field="organization_id"
-    )
-    user_email = models.ForeignKey(
-        User, on_delete=models.CASCADE, to_field="user_email"
-    )
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
     rating = models.IntegerField()
     comment = models.TextField(blank=True, null=True)
     active = models.BooleanField(default=True)
     created_at = models.DateTimeField(auto_now_add=True)
     modified_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return f"Review by {self.user.first_name} for {self.organization.organization_name}"
+        return f"Review by {self.user.get_full_name() if self.user else 'Anonymous'} for {self.organization.name}"
```

Comment:
I think this should be {self.organization.organization_name} because name is not a field in organization

Reply:
Will solve now